deployments/docker/examples/splunk/README.md

```diff
@@ -0,0 +1,17 @@
+# SignalFX agent Splunk Docker-compose example
+
+This example showcases how the agent works with Splunk Enterprise.
+
+The example runs as a Docker Compose deployment. The agent can be configured to send various metrics to Splunk Enterprise.
+
+Splunk is configured to receive data from the SignalFX Agent using the HTTP Event collector. To learn more about HEC, visit [our guide](https://dev.splunk.com/enterprise/docs/dataapps/httpeventcollector/).
+
+To deploy the example, open a terminal and in this directory type:
+```bash
+$> docker-compose up
+```
+:
+Splunk will become available on port 8000. You can login on [http://localhost:18000] with `admin` and `changeme`.
```

Comment:
change `Splunk will become available on port 8000.` to `Splunk will become available on port 18000.`

Reply:
Thanks good catch!